### PR TITLE
release notes for v1.0.1

### PR DESCRIPTION
**Pull request by pjfanning**

### CODE REVIEW

File: docs/src/main/paradox/release-notes/index.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 # 0. Release Notes
 
+## 1.0.1
+
+This is a bug fix release. There have also been some doc and build improvements.
+
+### Bug Fixes
+* Add optional throttle to handle excessive HTTP/2 non-data frames ([PR394](https://github.com/apache/incubator-pekko-http/pull/394))
+* Set the minimum required Pekko version to 1.0.0 (had been 1.0.1) ([PR328]https://github.com/apache/incubator-pekko-http/pull/328))
```
Comment (He-Pin) [inline docs/src/main/paradox/release-notes/index.md:9]: back to 1.0.0 , was 1.0.1

**Review by He-Pin — APPROVED** [review]
Lgtm overall

File: docs/src/main/paradox/release-notes/index.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
 # 0. Release Notes
 
+## 1.0.1
+
+This is a bug fix release. There have also been some doc and build improvements.
```
Comment (He-Pin) [inline docs/src/main/paradox/release-notes/index.md:5]: Need a diff link or a milestone to see what's changed.

**Review by He-Pin — CHANGES_REQUESTED** [review]

File: docs/src/main/paradox/release-notes/index.md (hunk 2 above)
Comment (pjfanning) [reply docs/src/main/paradox/release-notes/index.md:5]: we haven't provided that in any previous set of release notes

Comment (pjfanning) [reply docs/src/main/paradox/release-notes/index.md:5]: there is no release tag yet to use for a diff but users can use GitHub Compare for themselves

Comment (pjfanning) [reply docs/src/main/paradox/release-notes/index.md:5]: I have added #443 - this matches what we have in https://github.com/apache/incubator-pekko/blob/main/CHANGELOG.md

**Review by He-Pin — APPROVED** [review]
Nice and thanks

**Pull request merged**